docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform.md

```diff
@@ -0,0 +1,99 @@
+
+```javascript
+const parse = require('pg-connection-string').parse;
+const config = parse(process.env.DATABASE_URL);
+
+module.exports = () => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'bookshelf',
+      settings: {
+        client: 'postgres',
+        host: config.host,
+        port: config.port,
+        database: config.database,
+        username: config.user,
+        password: config.password,
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      },
+      options: {},
+    },
+  },
+});
+```
+
+Your application is now ready to deploy to DigitalOcean App Platform.
+
+## Deploying Strapi to DigitalOcean App Platform
+
+App Platform lets you deploy your application directly from a [GitHub](https://github.com) repo. [Gitlab](https://gitlab.com) is also supported.
+
+### Step 1. Log in to your [DigitalOcean account](https://cloud.digitalocean.com/login).
+
+### Step 2. Create a new "App" by clicking "Apps" in the "Create" dropdown
+
+### Step 3. Choose GitHub (or wherever you have your Strapi repo)
+
+### Step 4. Select your repository
+
+Choose your repository, your branch, and keep "Autodeploy code changes" checked if you want DigitalOcean to deploy every time you push to this GitHub branch.
+
+### Step 5. Configure your app
+
+Here you will configure how DigitalOcean App Platform deploys your Strapi app. You can leave most things default. The only things you need to change are shown below:
+
+- **Environment Variables**: Add `DATABASE_URL`: `${db.DATABASE_URL}`
+- **Build Command**: `NODE_ENV=production npm run build`
+- **Run Command**: `NODE_ENV=production npm start`
+
+### Step 6. Add a Database
+
+Click on the Add a Database button. You can create a development PostgreSQL database while testing your application. Alternatively, you can add a Managed Database that you have created.
+
+Name your database (default name is `db`). Whatever you name your database here is what you should use in the environment variables in Step 5 above. For instance, we name the database `db` and we use the environment variable value: `${db.DATABASE_URL}`
+
+Click "Next".
+
+### Step 7. Name your app
+
+Name your app. This will also change what domain your app will live on: `https://app-name.ondigitalocean.app`
+
+Select the region closest to you and your users. Static components are served on our global CDN.
+
+### Step 8. Choose your plan
+
+For prototype applications, you can choose the Basic plan. For applications that are expecting production traffic, you can choose the Pro plan. You will also see the pricing for your database that you have chosen.
+
+Choose your container size based on how much traffic you believe your app will have. It is a good practice to start on the smaller sizes, monitor the metrics of your app, and scale up as your app grows. App Platform allows us to scale vertically or horizontally with the click of a button.
```

Comment:
```suggestion
Choose your container size based on how much traffic you believe your app will have. It is a good practice to start on the smaller sizes, monitor the metrics of your app, and scale up as your app grows. App Platform allows DigitalOcean to scale vertically or horizontally with the click of a button.
```

Suggesting this because Strapi documentation is usually written from Strapi's point of view ;-)